src/examples/src/widgets/title-pane/Basic.tsx

```diff
@@ -1,8 +1,27 @@
 import { create, tsx } from '@dojo/framework/core/vdom';
 import TitlePane from '@dojo/widgets/title-pane';
+import icache from '@dojo/framework/core/middleware/icache';
 
-const factory = create();
+const factory = create({ icache });
 
-export default factory(function Basic() {
-	return <TitlePane title="Basic Title Pane" />;
+export default factory(function Basic({ middleware: { icache } }) {
+	const open = icache.get('open');
```

Comment:
If you use `getOrSet('open', false)`, you won't need to use `!!open` as it will always be either `true` or `false`

Reply:
good tip, thanks